tests/components/soft_navigations/api.test.js

```diff
@@ -281,33 +283,48 @@ test('.actionText and .setAttribute add attributes to ixn specifically', () => {
 
 // This isn't just an API test; it double serves as data validation on the querypack payload output.
 test('multiple finished ixns retain the correct start/end timestamps in payload', async () => {
-  softNavAggregate.ee.emit(`${INTERACTION_API}-get`, [0])
+  jest.useFakeTimers({ now: 0 })
+  softNavAggregate.ee.emit(`${INTERACTION_API}-get`, [jest.now()])
   let ixnContext = getIxnContext(mainAgent.interaction())
   ixnContext.associatedInteraction.nodeId = 1
   ixnContext.associatedInteraction.id = 'some_id'
   ixnContext.associatedInteraction.forceSave = true
-  softNavAggregate.ee.emit(`${INTERACTION_API}-end`, [200], ixnContext)
+  jest.advanceTimersByTime(200)
+  softNavAggregate.ee.emit(`${INTERACTION_API}-end`, [jest.now()], ixnContext)
 
-  softNavAggregate.ee.emit(`${INTERACTION_API}-get`, [300])
+  jest.advanceTimersByTime(100)
+  softNavAggregate.ee.emit(`${INTERACTION_API}-get`, [jest.now()])
   ixnContext = getIxnContext(mainAgent.interaction())
   ixnContext.associatedInteraction.nodeId = 2
   ixnContext.associatedInteraction.id = 'some_other_id'
   ixnContext.associatedInteraction.forceSave = true
-  softNavAggregate.ee.emit(`${INTERACTION_API}-end`, [500], ixnContext)
+  jest.advanceTimersByTime(200)
+  softNavAggregate.ee.emit(`${INTERACTION_API}-end`, [jest.now()], ixnContext)
 
-  softNavAggregate.ee.emit(`${INTERACTION_API}-get`, [700])
+  jest.advanceTimersByTime(200)
+  softNavAggregate.ee.emit(`${INTERACTION_API}-get`, [jest.now()])
   ixnContext = getIxnContext(mainAgent.interaction())
   ixnContext.associatedInteraction.nodeId = 3
   ixnContext.associatedInteraction.id = 'some_another_id'
   ixnContext.associatedInteraction.forceSave = true
-  softNavAggregate.ee.emit(`${INTERACTION_API}-end`, [1000], ixnContext)
+  jest.advanceTimersByTime(300)
+  softNavAggregate.ee.emit(`${INTERACTION_API}-end`, [jest.now()], ixnContext)
+  jest.useRealTimers()
+  jest.clearAllTimers()
```

Comment:
So I don't believe this actually fixes the underlying problem with real time being used to seek the fake-time ixn.
I also wouldn't mess with the faketimer set up for the rest of the file by switching back to real timer after this test.
--let's talk about this when you have a moment.

Reply:
reminder for shifting to this style: <img width="761" height="432" alt="image" src="https://github.com/user-attachments/assets/4f476851-761b-4d26-8132-7e760f4c3182" />